src/rpm_ostree/actor.rs

```diff
@@ -24,6 +24,8 @@ impl RpmOstreeClient {
 /// Request: stage a deployment (in finalization-locked mode).
 #[derive(Debug, Clone)]
 pub struct StageDeployment {
+    /// Whether to allow downgrades.
+    pub allow_downgrade: bool,
```

Comment:
Feels more like it'd be a property of the actor (e.g. a field of `RpmOstreeClient`) than a dynamic field from the message, though no strong opinion! I guess if Zincati learns to support `systemctl reload` in the future, it's easier this way.

Reply:
The remark about `reload` is interesting (I still don't know how to fit it in Zincati). The boolean setting itself has effects on both rpm-ostree and cincinnati logic, so I'd prefer to keep it in the main actor.

Reply:
Yup, SGTM.